conbench/entities/_entity.py

```diff
@@ -64,7 +64,7 @@ def to_float(value):
     return float(value) if value is not None else None
 
 
-T = TypeVar("T")
+T = TypeVar("T", bound="EntityMixin")
```

Comment:
This allows mypy to know that `cls` down below can use EntityMixin's methods.

Reply:
Thanks for thinking that through, reading docs